features/step_definitions/node.rb

```diff
@@ -140,6 +140,19 @@
   cb[tmpcb] = nil
 end
 
+Given /^I run( background)? commands on the nodes in the#{OPT_SYM} clipboard after scenario:$/ do |bg, cbname, table|
+  ensure_admin_tagged
```

Comment:
Perhaps we should add `ensure_destructive_tagged` ?
We can allow to run non-destructive ones in destructive mode, but we should not allow to run destructive ones in non-destructive mode.